Sources/swift-test-codecov/main.swift

```diff
@@ -160,11 +177,17 @@ extension StatsCommand {
     }
 
     func printMinimal(_ aggregateCoverage: Aggregate) {
+        print("")
         print(aggregateCoverage.formattedOverallCoveragePercent)
+        print("")
```

Comment:
I'd like to leave the spacing alone here. This has always been useful to display inline as "the coverage is $(swift-test-codecov ...)" -- Of course, your `numeric` type could be used pretty easily to reproduce the old behavior in this situation, but I still think "minimal" need not produce vertical padding.

Reply:
Fair enough. Reverting...